pkg/jobs/registry.go

```diff
@@ -1122,6 +1123,9 @@ func (r *Registry) stepThroughStateMachine(
 			defer jm.CurrentlyRunning.Dec(1)
 			err = resumer.Resume(resumeCtx, execCtx)
 		}()
+
+		r.MarkIdle(job, false)
```

Comment:
probably not needed since false is the default anyway?

Reply:
Oh sorry forgot to respond to this last time, this is to unmark its idleness after its been potentially marked idle during its `resumer.Resume(...)` lifetime.